src/components/where-to-buy/OnlineStores/OnlineStores.tsx

```diff
@@ -0,0 +1,30 @@
+import { SocialMediaList } from "@/constant/Links"
+import PlatformCard from "./PlatformCard"
+import { useTranslations } from "next-intl"
+
+export default function OnlineStores() {
```

Comment:
mobile versions: ui is not the same with figma; for example, padding on top too large for smaller phones, gap between each social on small devices

destop version: no extra space at the bottom like the design, use minheight to achieve the result